README.md

```diff
@@ -71,6 +71,28 @@ jobs:
     - run: flutter build web
 ```
 
+Build for Windows and upload artifact:
+
+```yaml
+  windows:
+    runs-on: windows-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-java@v1
+        with:
+          java-version: '12.x'
+      - uses: subosito/flutter-action@v1
+        with:
+          channel: dev
+      - run: flutter config --enable-windows-desktop
+      - run: flutter create .
```

Comment:
there should be no use for `flutter create .`.
it may only create the desktop platform folders. But if they are not there already, the project probably is not intended to be run on desktop.

May I suggest a `flutter pub get`, like in the other example, instead? (it implicitly gets run by `flutter create`)

Reply:
Yes you are right. I used that because I did not specifically targeted windows, only built cause it could be build. I do not know what have changed since Flutter 2.0 as I am not using flutter since January.